tests/src/main/java/com/hazelcast/stabilizer/tests/slow/SlowOperationMapTest.java

```diff
@@ -0,0 +1,193 @@
+package com.hazelcast.stabilizer.tests.slow;
+
+import com.hazelcast.core.IMap;
+import com.hazelcast.instance.Node;
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.logging.Logger;
+import com.hazelcast.map.MapInterceptor;
+import com.hazelcast.spi.impl.InternalOperationService;
+import com.hazelcast.stabilizer.Utils;
+import com.hazelcast.stabilizer.test.TestContext;
+import com.hazelcast.stabilizer.test.TestRunner;
+import com.hazelcast.stabilizer.test.annotations.RunWithWorker;
+import com.hazelcast.stabilizer.test.annotations.Setup;
+import com.hazelcast.stabilizer.test.annotations.Teardown;
+import com.hazelcast.stabilizer.test.annotations.Verify;
+import com.hazelcast.stabilizer.test.annotations.Warmup;
+import com.hazelcast.stabilizer.test.utils.ExceptionReporter;
+import com.hazelcast.stabilizer.tests.helpers.KeyLocality;
+import com.hazelcast.stabilizer.tests.helpers.KeyUtils;
+import com.hazelcast.stabilizer.worker.OperationSelector;
+import com.hazelcast.stabilizer.worker.tasks.AbstractWorkerTask;
+
+import java.lang.reflect.Method;
+import java.util.Collection;
+import java.util.Random;
+import java.util.concurrent.atomic.AtomicLong;
+
+import static com.hazelcast.stabilizer.tests.helpers.HazelcastTestUtils.getNode;
+import static com.hazelcast.stabilizer.tests.helpers.HazelcastTestUtils.getOperationCountInformation;
+import static com.hazelcast.stabilizer.tests.helpers.HazelcastTestUtils.waitClusterSize;
+import static com.hazelcast.stabilizer.utils.ReflectionUtils.getMethodByName;
+import static com.hazelcast.stabilizer.utils.ReflectionUtils.invokeMethod;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class SlowOperationMapTest {
```

Comment:
Can you add a line of comment with what this does and how it works? Using the interceptor to cause delays --> smart.